boards/arm/nrf52810_pca20045/nrf52810_pca20045.dts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/dts-v1/;
+#include <nordic/nrf52810_qfaa.dtsi>
+
+/ {
+	model = "Nordic PCA20045";
+	compatible = "nordic,nrf52810-qfaa",
+		     "nordic,nrf52810";
+
+	chosen {
+		zephyr,console = &uart0;
+#ifdef CONFIG_MCUMGR_SMP_UART
+		zephyr,uart-mcumgr = &uart0;
+#endif
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+};
+
+&adc {
+	status ="ok";
+};
+
+&gpiote {
+	status ="ok";
+};
+
+&gpio0 {
+	status ="ok";
+};
+
+&uart0 {
+	status = "ok";
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	tx-pin = <35>;
```

Comment:
pins are obviously wrong

Reply:
Also, I had the impression the 52810 has only uarte

Reply:
Also, haven't we moved to using `	status = "okay";`

Reply:
> wrong

How about this comment, @pdunaj ? Is this now fixed?

Reply:
Yes, all 3 are fixed.